packages/mui-material-next/src/ListItemButton/ListItemButton.test.js

```diff
@@ -99,7 +99,9 @@ describe('<ListItemButton />', () => {
       expect(!!link).to.equal(true);
     });
 
-    it('should rendered as specifying component', () => {
+    // TODO v7: support hostElementName prop
+    // eslint-disable-next-line mocha/no-skipped-tests
+    it.skip('should rendered as specifying component', () => {
```

Comment:
And one below. I know it's not your code but it caught my eye and we may improve it by the way.

```suggestion
    it.skip('should be rendered as the specified component', () => {
```